articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to create a Availability Set | UKCloud Ltd
+description: Creating a Availability Set in UKCloud for Microsoft Azure
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: 
+toc_sub3:
+toc_sub4:
+toc_title: Create a Availability
+toc_fullpath: Users/How To/azs-how-create-availability-set.md
+toc_mdlink: azs-how-create-availability-set.md
+---
+
+# Overview
+
+In this guide, you learn how to increase the availability and reliability of your Virtual Machine solutions on Azure Stack using a capability called **Availability Sets**. Availability sets ensure that the VMs you deploy on Azure Stack are distributed across multiple isolated hardware nodes in a cluster. Doing this ensures that if a hardware or software failure within Azure Stack happens, only a subset of your VMs are impacted and that your overall solution remains available and operational.
```

Comment:
In this guide, you'll learn...
virtual machine doesn't need uppercase V and M
Don't need the second that in the last sentence:
Doing this ensure that, if a hardware or software failure within Azure Stack happens, only a subset of your VMs are impacted and your overall solution remains available and operational.

Reply:
done